ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2297,3 +2297,14 @@ def get_sssd_version(host):
     """Get sssd version on remote host."""
     version = host.run_command('sssd --version').stdout_text.strip()
     return parse_version(version)
+
+
+def enable_encryption(host, domain_name):
```

Comment:
@amore17 , @flo-renaud
Can we add ansible play for this in https://github.com/freeipa/freeipa-pr-ci/tree/master/ansible/roles/windows/common/tasks to enable AES on ad vms ?
This way we can avoid calling this every test suite

Reply:
The step is needed only when there is a subdomain, in this case https://github.com/freeipa/freeipa-pr-ci/blob/master/ansible/roles/windows/ad-subdomain/tasks/main.yml would be a better fit IMO.

Reply:
I think we need to enable AES krb5 communication on every AD vms as it has affected kinit with every, so we should add in common task.  We have added this on every AD vms in downstream.

Reply:
Added : https://github.com/freeipa/freeipa-pr-ci/pull/380